src/policy/copyspace.rs

```diff
@@ -126,6 +127,10 @@ impl<VM: VMBinding> crate::policy::gc_work::PolicyTraceObject<VM> for CopySpace<
         copy: Option<CopySemantics>,
         worker: &mut GCWorker<VM>,
     ) -> ObjectReference {
+        assert!(
```

Comment:
This is performance critical code. So use `debug_assert` or `static_assertions` instead.